cudatoolkit-dev/scripts/cuda_activate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash 
+
+cd $CONDA_PREFIX/bin
+find . -type l -exec unlink {} \;
+ln -s $CONDA_PREFIX/pkgs/cudatoolkit-dev/bin/* .
```

Comment:
I think you need to link everything that isn't in the `info/` dir, not just what is in the bin dir.

Reply:
The following is the list of directories that get shipped with cudatoolkit. 
```bash
abanihirwe@yellowstone:/usr/local/cuda$ ls
bin     include  libnsight         nvml       samples  tools
doc     jre      libnvvp           nvvm       share    version.txt
extras  lib64    nsightee_plugins  pkgconfig  src
```
Are you saying that we need to link all these other directories not the bin directory only? Let me know If I am getting this right.. The usual cudatoolkit installation just adds the bin dir to the `$PATH`. This is the reason why I was linking bin dir only.

Reply:
> Are you saying that we need to link all these other directories not the bin directory only?

Yes, exactly.  The $PATH only matters for executables, but other programs will have to find libraries, MAN files, etc.  This is why we have to link everything.